### PR TITLE
Feat/463 vega transaction component links to block explorer

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes [#463]

# Description ℹ️

In order to make the UX around being able to see and understand transactions made in the console, we need to have links for all transactions to the block explorer.

# Demo 📺

image
# Technical 👨‍🔧

- [x]  Links take you to the view of that transaction in the block explorer (opens another tab)

### CODE REVIEW

File: libs/wallet/src/vega-order-transaction-dialog/vega-order-transaction-dialog.tsx [hunk 1]
```diff
@@ -74,7 +76,15 @@ export const VegaOrderTransactionDialog = ({
       >
         {transaction.txHash && (
           <p data-testid="tx-hash" className="break-all">
-            {t(`Tx hash: ${transaction.txHash}`)}
+            Tx hash: &nbsp;
+            <a
```
Comment (dexturr) [inline libs/wallet/src/vega-order-transaction-dialog/vega-order-transaction-dialog.tsx:80]: chore (blocking): can we please add a test to verify this?

File: apps/trading/lib/config/env.ts [hunk 2]
```diff
@@ -17,5 +17,9 @@ export const ENV = {
     process.env['NX_TRADING_SENTRY_DSN']
   ),
   envName: windowOrDefault('NX_VEGA_ENV', process.env['NX_VEGA_ENV']),
-  vegaUrl: windowOrDefault('NX_VEGA_URL', process.env['NX_VEGA_URL']),
+  vegaUrl: windowOrDefault('NX_VEGA_URL', process.env['X_VEGA_URL']),
+  explorerUrl: windowOrDefault(
```
Comment (notbot00) [inline apps/trading/lib/config/env.ts:21]: chore (blocking): i think it is confusing to have this duplication in both libs/environment and here. looks like you are using the env hook to get the value, so this here can be deleted imo.

Comment (dexturr) [reply apps/trading/lib/config/env.ts:21]: This was my mistake when we were talking about it. My bad @MadalinaRaicu

File: libs/wallet/src/vega-order-transaction-dialog/vega-order-transaction-dialog.tsx (hunk 1 above)
Comment (MadalinaRaicu) [reply libs/wallet/src/vega-order-transaction-dialog/vega-order-transaction-dialog.tsx:80]: sure, pushing it now

File: apps/trading/lib/config/env.ts [hunk 3]
```diff
@@ -17,5 +17,5 @@ export const ENV = {
     process.env['NX_TRADING_SENTRY_DSN']
   ),
   envName: windowOrDefault('NX_VEGA_ENV', process.env['NX_VEGA_ENV']),
-  vegaUrl: windowOrDefault('NX_VEGA_URL', process.env['NX_VEGA_URL']),
+  vegaUrl: windowOrDefault('NX_VEGA_URL', process.env['X_VEGA_URL']),
```
Comment (MadalinaRaicu) [inline apps/trading/lib/config/env.ts:20]:
```suggestion
  vegaUrl: windowOrDefault('NX_VEGA_URL', process.env['NX_VEGA_URL']),
```

**Review by notbot00 — APPROVED** [review]

**Review by dexturr — APPROVED** [review]

**Pull request merged**